lib/generators/solidus_stripe/install/templates/app/views/checkouts/payment/_stripe.html.erb

```diff
@@ -1,22 +1,47 @@
-<% payment_element_options = {
-  mode: 'payment',
-  amount: SolidusStripe::MoneyToStripeAmountConverter.to_stripe_amount(
-    current_order.display_order_total_after_store_credit.money.fractional,
-    current_order.currency,
-  ),
-  currency: current_order.currency.downcase,
-  paymentMethodCreation: 'manual',
-  captureMethod: payment_method.auto_capture? ? 'automatic' : 'manual',
-  # Fully customizable with appearance API.
-  # https://stripe.com/docs/elements/appearance-api
-  # appearance: {},
-} %>
+<%
+  bill_address = current_order.bill_address
+  options = {
+    elementsInitialization: {
+      mode: 'payment',
+      amount: SolidusStripe::MoneyToStripeAmountConverter.to_stripe_amount(
+        current_order.display_order_total_after_store_credit.money.fractional,
+        current_order.currency,
+      ),
+      currency: current_order.currency.downcase,
+      paymentMethodCreation: 'manual',
+      captureMethod: payment_method.auto_capture? ? 'automatic' : 'manual',
+      # paymentMethodTypes: ['card', 'sepa_debit'],
```

Comment:
nit: Thoughts on adding a short comment above this line indicating it's an example of customizing the method types list?